<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramWidget</class>
 <widget class="QWidget" name="ProgramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDial" name="volumeDial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>255</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="volumeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="panningDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="initButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </widget>
  <widget class="QLabel" name="panningLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Panning</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="storeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Store</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>325</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="slotSpin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>104</number>
   </property>
  </widget>
  <widget class="QLabel" name="fileSystemLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File system:</string>
   </property>
  </widget>
  <widget class="QLabel" name="internalLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stealingButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stealing</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="programLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Program:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>106</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>openButton</zorder>
  <zorder>volumeDial</zorder>
  <zorder>saveButton</zorder>
  <zorder>volumeLabel</zorder>
  <zorder>panningDial</zorder>
  <zorder>initButton</zorder>
  <zorder>panningLabel</zorder>
  <zorder>loadButton</zorder>
  <zorder>slotSpin</zorder>
  <zorder>fileSystemLabel</zorder>
  <zorder>internalLabel</zorder>
  <zorder>nameLine</zorder>
  <zorder>stealingButton</zorder>
  <zorder>storeButton</zorder>
  <zorder>programLabel</zorder>
  <zorder>muteButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
